### ISSUE TITLE
Add Ability to Draw Dots

### ISSUE THREAD

**Issue by yeliknewo** [PistonDevelopers/graphics#1025]
I am making a "game" that is like the game of life. I don't have a fast enough GPU to render a full screen of one pixel rectangles quickly. If I drew the rectangles as dots could I get a noticeable performance increase? If so is anyone else interested in this? Should I implement this?

**Comment by bvssvni** [comment]
Another way to do this is to change the pixels in an image and then update a texture. See https://github.com/PistonDevelopers/piston-examples/blob/master/src/paint.rs.

**Comment by yeliknewo** [comment]
Is there a speed difference between drawing a texture where you update every pixel of the texture every update and drawing each pixel individually every frame?

**Comment by bvssvni** [comment]
@yeliknewo Yes. Updating the pixels of an image can be done much faster since you do not have to send data to the GPU until the texture is updated.
